packages/esm-form-entry-app/src/app/form-data-source/form-data-source.service.ts

```diff
@@ -3,9 +3,11 @@ import { Injectable } from '@angular/core';
 import { take, map, tap } from 'rxjs/operators';
 import { Observable, Subject } from 'rxjs';
 
+import { FHIRResource } from '@openmrs/esm-framework';
```

Comment:
Always use `import type` if we're just importing types from a library... It allows the declaration to be erased.

```suggestion
import type { FHIRResource } from '@openmrs/esm-framework';
```

Reply:
Yeah *usually* the declaration gets erased anyway, but (1) it makes sure that it definitely *will* be erased (even though TS might think there are side-effects etc.) and (2) it's also a good hint for the reader that the imported object it really just a type.

For things like `interface` and `type` the story is anyway simple, but if we'd import a `class` then such a `type` import would also guard us from actually using it in actual code; which can be a good thing for a couple of reasons.

Anyway - just wanted to emphasize that this suggestion is great and should be followed.